gst/nnstreamer/tensor_split/gsttensorsplit.c

```diff
@@ -645,7 +645,12 @@ gst_tensor_split_get_property (GObject * object, guint prop_id,
         strings = (gchar **) g_ptr_array_free (arr, FALSE);
         p = g_strjoinv (":", strings);
         g_free (strings);
-        strv = g_strjoin (",", strv, p, NULL);
+        if (i > 0) {
+          strv = g_strjoin (",", strv, p, NULL);
```

Comment:
Can the value of `self->tensorseg->len` be 2 or higher?
In that case, I think strv is overwritten by g_strjoin() without freeing the old memory of strv (g_strjoin requires the caller to free the memory of the returned string).

Reply:
Good catch. Tensorseg's len can be up to 16.